bin/blueoak-server.js

```diff
@@ -3,10 +3,21 @@
  * Copyright (c) 2015-2016 PointSource, LLC.
  * MIT Licensed
  */
+var parseArgs = require('minimist');
 var server = require('../');
 
+// parse arguments
+var argv = parseArgs(process.argv.slice(2));
+
+// convert mocks from CSV into an array
+var mocks = argv.mocks || argv.m;
+if (mocks) {
+    mocks = mocks.split(',');
+}
+
 server.init({
-    appDir: process.cwd()
+    appDir: process.cwd(),
+    mocks: mocks
```

Comment:
maybe we should have a different script/invocation that supports mocks instead of adding testing support directly into the main command ...
say `bin/bos-mock.js`